src/main/scala/cognite/spark/v1/PushdownUtilities.scala

```diff
@@ -8,7 +8,7 @@ import org.apache.spark.sql.sources._
 import org.apache.spark.sql.types.{DataTypes, StructField, StructType}
 import scala.util.Try
 
-case class DeleteItem(id: Long)
+case class DeleteItem(id: Long, ignoreUnknownIds: Option[Boolean] = None)
```

Comment:
I don't think we need to bother with making this optional... can't think of any use case for when we'd like deletes on missing items to give errors, and the code will be so much simpler if we just always use this flag in the Spark data source.

Reply:
We can support it as an option on the relation, `.option("ignoreUnknownIds", "false")` (with a default of true).

Reply:
why should the default be true? In the scala sdk it defaults to false, plus that was the behavior before this was added.

Reply:
In the SDK(s) we try to stay as close to the API itself as possible, where ignoreUnknownIds is false by default.

For users of the Spark data source we expect that almost always they'd rather not have their job fail if something they're trying to delete has already been deleted. We're adding this feature to the Spark data source *because* users have been disappointed when deletes fail because something has already been deleted. ;)